### PR TITLE
Create indel_locus_extractor.py

### PR DESCRIPTION
**Pull request by hopedisastro**
This script extracts the coordinates of every indel recorded in a cohort's Hail matrix table. 

Trying out an experiment to see if I can use indels as a proxy for a potential STR by intersecting these output indel coordinates with my STR catalog.

### CODE REVIEW

**Comment by hopedisastro** [comment]
I have run `black` on my end and it doesn't change the script, so not sure why the lint check is failing at `black`.

File: str/indel_locus_extractor.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    b = get_batch()
+    hail_job = b.new_python_job(name=f'Hail query filter for indels')
+    hail_job.image(config['workflow']['driver_image'])
+    hail_job.storage('20G')
+    hail_job.cpu(4)
+    gcs_output_path = output_path(f'indels.tsv', 'analysis')
+    hail_job.call(indel_coordinate_extractor, file_path, gcs_output_path)
+
+    b.run(wait=False)
```
Comment (MattWellie) [inline str/indel_locus_extractor.py:56]:
```suggestion
    gcs_output_path = output_path(f'indels.tsv', 'analysis')
    indel_coordinate_extractor(file_path, gcs_output_path)
```

You're already running this inside the driver image in hail, you should be fine running this without a second layer of batching

File: str/indel_locus_extractor.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+import hail as hl
+import click
+
+from cpg_utils.config import get_config
+from cpg_utils.hail_batch import output_path, init_batch
+from cpg_workflows.batch import get_batch
+
+
+config = get_config()
```
Comment (MattWellie) [inline str/indel_locus_extractor.py:22]:
```suggestion
import hail as hl
import click

from cpg_utils.hail_batch import output_path, init_batch
from cpg_workflows.batch import get_batch

```

If you don't create a nested batch job you don't need the config here

**Comment by hopedisastro** [comment]
Ah, I had to update my `black` version!

**Review by MattWellie — APPROVED** [review]

**Pull request merged**